test/Customer-test.js

```diff
@@ -35,9 +29,19 @@ describe('Customer', function() {
     expect(customer2.name).to.equal("Rocio Schuster");
   });
 
-  it('Should be able to find customers bookings by customer Id', function() { 
-   
-    // expect(customer1.findBookings()).to.include.members([booking1, booking2])
+  it('Should be able to rearrange dates to another format', function() { 
+    customer1.pastReservations.push({"id":"5fwrgu4i7k55hl6x8","userID":1,"date":"2022/01/11","roomNumber":20,"roomServiceCharges":[]})
+    customer1.rearrangeDate();
+
+    expect(customer1.pastReservations[0].date).to.deep.equal("01/11/2022")
+  });
+
+  it('Should be able to calculate the room total', function() { 
+    customer1.pastReservations.push({"id":"5fwrgu4i7k55hl6x8","userID":1,"date":"2022/01/11","roomNumber":20,"roomServiceCharges":[], "costOfRoom": 207.67})
```

Comment:
It's so cool that you're pushing in a whole object, creating one source of truth to pull information from.